src/main/java/com/coherentsolutions/training/automation/web/sirbu/pageobjects/HomePage.java

```diff
@@ -0,0 +1,60 @@
+package com.coherentsolutions.training.automation.web.sirbu.pageobjects;
+
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+
+public class HomePage extends BasePage {
+
+    private static final By LOGGED_IN_LOCATOR = By.className("logged-in");
+
+    @FindBy(xpath = "//div[@class='panel header']//button[@type='button']")
+    private WebElement myAccountButton;
+    @FindBy(xpath = "//div[@aria-hidden='false']//a[normalize-space()='Sign Out']")
+    private WebElement logoutLink;
+
+    @FindBy(className = "logged-in")
+    private WebElement welcomeMessage;
+
+    @Override
+    public boolean isOpened() {
+        return isLoggedIn();
+    }
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+
+    public boolean isLoggedIn() {
+        try {
+            return welcomeMessage.isDisplayed();
+        } catch (NoSuchElementException e) {
+            return false;
+        }
```

Comment:
wrap this logic up to a separate method
just create a utility class - WebElementUtils and place this method there